Dockerfile

```diff
@@ -0,0 +1,101 @@
+FROM ros:humble-ros-base-jammy AS base
+
+# Install basic dev tools (And clean apt cache afterwards)
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive \
+        apt-get -y --quiet --no-install-recommends install \
+        # Download tool
+        curl \
+        # Install Cyclone DDS ROS RMW
+        ros-"$ROS_DISTRO"-rmw-cyclonedds-cpp \
+    && rm -rf /var/lib/apt/lists/*
+
+# Setup Dataspeed apt
+RUN curl -sSL https://bitbucket.org/DataspeedInc/ros_binaries/raw/master/dataspeed.key -o /usr/share/keyrings/dataspeed-archive-keyring.gpg \
+    && sh -c 'echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/dataspeed-archive-keyring.gpg] http://packages.dataspeedinc.com/ros2/ubuntu \
+    $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | sudo tee /etc/apt/sources.list.d/ros2-dataspeed-public.list > /dev/null'
+
+# Setup Dataspeed rosdep
+RUN sh -c "echo \"yaml http://packages.dataspeedinc.com/ros2/ros-public-\$ROS_DISTRO.yaml \$ROS_DISTRO\" \
+    > /etc/ros/rosdep/sources.list.d/30-dataspeed-public-\$ROS_DISTRO.list"
+
+
+# Install basic dev tools (And clean apt cache afterwards)
+RUN apt-get update \
+    && rosdep update \
+    && DEBIAN_FRONTEND=noninteractive \
+        apt-get -y --quiet --no-install-recommends install \
+        # Dataspeed Ford Drive-By-Wire
+        ros-"$ROS_DISTRO"-dbw-ford \
+        # Dataspeed Power Distribution System
+        # ros-noetic-dataspeed-pds-can \
```

Comment:
Does this work? Or leftover copy-paste?

Reply:
Left ofver-copy, I couldn't find the ROS 2 version of that pkg, last [commit](https://bitbucket.org/DataspeedInc/dataspeed_pds/src/master/) was over 4 years ago. Will remove it

Reply:
Done